src/components/ShowPage.js

```diff
@@ -0,0 +1,21 @@
+import React, { useState, useEffect} from "react"
+import AnimalInfo from "./AnimalInfo"
+
+const ShowPage = (props) =>  {
+    
+    const [pet, setPet] = useState({})
+    const petId = props.match.params.id 
+
+    useEffect(()  =>  {
+        fetch(`/api/v1/pets/${petId}`)
+        .then(response  => response.json())
```

Comment:
please include error handling in your fetches. 
You can use any of the examples where we check if the response status is ok and use catch if it isn't